servicex/templates/app/configmap.yaml

```diff
@@ -72,6 +72,10 @@ data:
 
     TRANSFORMER_NAMESPACE="{{ .Release.Namespace }}"
 
+    {{ if .Values.transformer.priorityClassName }}
+    TRANSFORMER_PRIORITY_CLASS="{{ .Values.transformer.priorityClassName }}"
```

Comment:
A default needs to be added to Servicex/values.yaml and documented in docs/deployment/reference.md